README.md

```diff
@@ -91,3 +91,25 @@ Results will be an object representing a table of data. It will have the followi
 Names: The names of the columns of the table. The first name will be t, identifying the time step column. The following names will identify the state and the age group being counted in that column. The age group will be identified in square brackets. E.g. the "S[1]" column represents the count for susceptables in the first age group.
 
 Y: A 2D array representing the rows of the table. The first dimension will depend on the time steps and resolution of the model. The second dimension will be the same size as "Names".
+
+#### beta and Rt
+
+The model is parameterised with a country specific `beta` value.
+
+To translate Rt values into beta values, you can divide them by the country
```

Comment:
Way out of my lane here, but in sircovid discussions about beta vs Rt there was some issues about accounting for depletion of susceptibles - is that somehow accounted for here?

Reply:
I've let OJ know, he'll update beta estimates upstream if it's relevant.